EmpleoDotNet/Controllers/HomeController.cs

```diff
@@ -14,11 +14,33 @@ public class HomeController : EmpleoDotNetController
     {
         public ActionResult Index()
         {
-            ViewBag.SearchViewModel = new JobOpportunitySearchViewModel {
+            var allJobOpportunities = _jobOpportunityRepository.GetAllJobOpportunities();
+
+            var model = new HomeViewModel();
+
+            var jobCount = allJobOpportunities.Count;
+
+            var visits = allJobOpportunities.Sum(x => x.ViewCount);
+
+            var companyCount = (from jo in allJobOpportunities
```

Comment:
I feel good with this linq form because is more easy read when you need group by